### DISCUSSION TITLE
Can you stretch images?

### DISCUSSION THREAD

**Discussion by Stingaaa** [cogentcore/core#1308] [Q&A] [answered]
Is there a way to stretch images unproportionally?
I looked at the Image struct itself and found that it has a Bounds function, which from my understanding gets set when reading the image source, which the framework uses the grow/ shrink the image proportionally based on the given widget size.
Therefore I was wondering if there is an option how I can change the Bounds dimensions (if possible).
If this does not work, is there another solution or should I simply a) choose a fitting image source or b) make a manipulator function which changes the resoltion/ proportion of the image source before calling the OpenFS function?

**Comment by kkoreilly** [comment, marked as answer]
You can try changing `s.ObjectFit` to something such as `styles.FitFill`. You can also modify `s.Min` in conjunction with that to override the default size. If that doesn't work for your use case, please let me know.

**Reply by Stingaaa** [reply]
Thank you very much for the reply, I tried s.ObjectFit and it worked immediately.
